README.md

```diff
@@ -44,21 +46,34 @@ The middleware writes the data returned by the Cloud Health module as JSON, and
   let healthcheck = new health.HealthChecker();
   ```
   
-3. Register a Liveness endpoint:
+3. Register a separate Liveness endpoint:
 
   ```js
-  app.use('/health', health.LivenessEndpoint(healthcheck))
+  app.use('/live', health.LivenessEndpoint(healthcheck))
   ```
   If no livessness checks are registered, this will report `200 OK` and `UP`.
     
-4. Register a readiness endpoint:
+4. Register a separate readiness endpoint:
 
   ```js
   app.use('/ready', health.ReadinessEndpoint(healthcheck))
   ```
   If no readiness checks are registered, this will report `200 OK` and `UP`.
+  
+5. Register a combined health endpoint:
+
+  ```js
+  app.use('/health', health.ReadinessEndpoint(healthcheck))
+  ```
+  If no readiness or liveness checks are registered, this will report `200 OK` and `UP`.
+
+For information on how to register startup, readiness, liveness and shutdown checks, see the [Cloud Health documentation](https://github.com/CloudNativeJS/cloud-health/blob/master/README.md).
+
+#### Health, Liveness and Readiness endpoints
+
+The difference between liveness and readiness endpoints is the purpose purpose: readiness should be used to denote whether an application is "ready" to receive requests, and liveness should be used to denote whether an application is "live" (vs. in a state where it should be restarted.
 
-For information on how to register startup, liveness and shutdown checks, see the [Cloud Health documentation](https://github.com/CloudNativeJS/cloud-health/blob/master/README.md).
+The health endpoint is designed for cloud technologies, such as cloud Foundry, that only support a single endpoint for both liveness and readiness checking.
```

Comment:
are you talking about the combined health check here,
possible suggestion:
The combined health check is designed for ...

Reply:
added "combined"